src/main/java/com/jamierf/dropwizard/debpkg/config/DebConfiguration.java

```diff
@@ -34,4 +41,8 @@ public DebConfiguration setMaintainer(final String maintainer) {
         this.maintainer = maintainer;
         return this;
     }
+    
+    public void setName(String name) {
+		this.name = name;
```

Comment:
The formatting looks a bit off here.

Also can we use the Builder pattern to be consistent with the other setters? (i.e. `public DebConfiguration setName(final String name)`).